### DISCUSSION TITLE
the 300c path confusion

### DISCUSSION THREAD

**Discussion by jxer** [mriscoc/Ender3V2S1#948] [Q&A] [answered]
so I ordered a epcos thermistor, and replaced my heatbreak on my e3v2.

i looked in the configuration.h and thermistor charts and see the extended temp table

I assume the c104 is due to concerns about people trying to raise above 260 without the above upgrades

my confusion

is this part added in the begin gcode section of cura? that makes sense, the m500 comment doesn't please help

**Comment by mriscoc** [comment]
You don't need to put the C104 in the slicer start script, you can send it one time and save with an M500. The best place to put a C104 is inside a [configuration gcode file](https://github.com/mriscoc/Ender3V2S1/wiki/Configuration-files).

C104 was designed to use higher temperatures in a modified printer without the need to recompile the firmware. But if you are going to do a custom compile, the proper way is changing the `.h` configuration files.

**Reply by jxer** [reply]
does it only save between firmware reloads? if I reload another one of your newer firmwares, do I need to run the c104 m500 commands again? I believe I do

**Reply by mriscoc** [reply]
Whenever you upgrade or downgrade the firmware, it is necessary to re-run all setup operations.

**Reply by jxer** [reply]
so just to close this out, if I compile myself, configuration.h line 630 should be changed to 315 from 275, no other changes are needed?

**Reply by mriscoc** [reply, marked as answer]
That's right

https://github.com/mriscoc/Ender3V2S1/blob/b4ee2b351e69672068911e87474394a35f3ce448/Marlin/Configuration.h#LL630C12-L630C12

**Reply by jxer** [reply]
Appreciate the Help on this!

**Comment by mriscoc** [comment]
If you are using the custom G-code C104, look at: https://github.com/mriscoc/Ender3V2S1/issues/957
